src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/TaskLogsTab.php

```diff
@@ -31,36 +33,50 @@ public function resolveTaskAction()
             $this->resolveTasks(
                 [
                     (int) $_GET['selected'],
-                ]
+                ],
+                $this->getResolveAction(),
             );
         }
-        wp_redirect(remove_query_arg(['selected', 'action']));
+        $this->clearArgs();
     }
 
     public function resolveTasksAction()
     {
-        if (array_key_exists('selected', $_POST)) {
+        if (array_key_exists('selected', $_POST) && array_key_exists('resolveAction', $_POST)) {
+            $resolveAction = $_POST['resolveAction'];
```

Comment:
I think it's just a row action

```suggestion
            $rowAction = $_POST['rowAction'];
```

Reply:
Renamed keys and variables as suggestion stated.